infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/NodeSelectorProvisioner.java

```diff
@@ -29,12 +26,8 @@ public class NodeSelectorProvisioner implements ConfigurationProvisioner {
   private final Map<String, String> nodeSelectorAttributes;
 
   @Inject
-  public NodeSelectorProvisioner(
```

Comment:
why can't we completely remove the NodeSelector?

Reply:
We can remove it. It is managed by operator.

Reply:
Sounds good, I can remove it completely.